runelite-client/src/test/java/net/runelite/client/plugins/cluescrolls/clues/EmoteClueTest.java

```diff
@@ -24,14 +24,59 @@
  */
 package net.runelite.client.plugins.cluescrolls.clues;
 
-import static org.junit.Assert.assertNull;
+import net.runelite.api.Item;
+import static net.runelite.api.ItemID.*;
+import static org.junit.Assert.*;
+
+import net.runelite.client.plugins.cluescrolls.clues.item.ItemRequirement;
```

Comment:
Import blocks should be without blank lines.
```suggestion
import net.runelite.api.Item;
import net.runelite.client.plugins.cluescrolls.clues.item.ItemRequirement;
import static net.runelite.api.ItemID.*;
import static org.junit.Assert.*;
```